src/server/server.js

```diff
@@ -46,8 +48,11 @@ app.use(expressWinston.logger({
 app.use(helmet());
 app.use(session(sessionConfig(conf)));
 app.use(Express.static(__dirname + '/../public', { maxAge: '365d' }));
+const metricsBundle = promBundle({ autoregister: false });
+app.use(metricsBundle);
 
 // routes
+app.use('/metrics', trustedNetworks, metricsBundle.metricsMiddleware);
```

Comment:
That's the idea!  It's not very important with the minimal set available at the moment, but likely will be eventually.